src/common/schedulers/slurm_commands.py

```diff
@@ -258,6 +258,13 @@ def get_partition_info(command_timeout=DEFAULT_GET_INFO_COMMAND_TIMEOUT, get_all
     ]
 
 
+def resume_powering_down_nodes():
+    """Resume nodes that are powering_down so that are set in power state right away."""
+    log.info("Resuming powering down nodes.")
+    powering_down_nodes = _get_slurm_nodes(states="powering_down")
+    update_nodes(nodes=powering_down_nodes, state="resume", raise_on_error=False)
```

Comment:
Do we want to reset nodeaddr and nodehostname here in case they are not reset and add a reason for the command: `reset_nodes(nodes=powering_down_nodes, state="resume", reason="Resuming powering down nodes.", raise_on_error=False)`
Although in the cases we saw about nodes in `IDLE+CLOUD+POWERING_DOWN`, nodeaddr and nodehostname are already reset